napari/layers/tracks/_track_shader.py

```diff
@@ -0,0 +1,162 @@
+from typing import List, Union
```

Comment:
Can this file be moved to `_vispy/_vispy_tracks_shader.py`, we're trying to keep all our code with vispy imports together there

Reply:
Will do. Much better location for it!